app/src/js/reducers/index.js

```diff
@@ -57,7 +57,7 @@ export const reducers = {
   reconciliationReports,
   recoveryStatus,
   sorts,
-  locationQueryParams,
+  locationQueryParams
```

Comment:
Why do you remove the trailing comma? Trailing commas are preferred in objects.

Reply:
I put the comma back. The reason why I removed it is because it is not necessary. This is the way I have been trained.